docs/sphinx/source/user_guide/index.rst

```diff
@@ -1,20 +1,34 @@
+.. _user_guide:
+
 ==========
 User Guide
 ==========
 
+This user guide is an overview and explains some of the key features of pvlib.
 
 .. toctree::
+   :caption: Getting started
    :maxdepth: 2
-
+   
    package_overview
    installation
    introtutorial
+   
+.. toctree::
+   :caption: Advanced usage
```

Comment:
I think "Advanced usage" may be undeservedly intimidating for some readers.  How about "Modeling topics" instead?